src/pyTRLCConverter/plantuml.py

```diff
@@ -73,8 +95,100 @@ def is_plantuml_file(self, diagram_path):
 
         return is_valid
 
-    def generate(self, diagram_type, diagram_path, dst_path):
-        """Generate image.
+    def _make_server_url(self, diagram_type: str, diagram_path: str) -> str:
```

Comment:
Its an open question whether we shall use return types or whether to use types for parameters too. But if we do it once, we should do it everywhere.

Reply:
I'm fully in favor of type hinting as part of documentation (PEP484). For me it improves readability and I always try to add them. (I fear that the the JS to TS story repeats an why need to lean a typeson  language dialect at some point in time.)

Reply:
Waiting for response. If type hints are recomended moving forward, it should be a dedicated commit to ass them in other places.

Reply:
To avoid blocking this PR, lets go ahead, but open a separate issue for clarification.

Reply:
See #9